client/app/components/Feed/Search/Search.jsx

```diff
@@ -1,12 +1,25 @@
-var React = require('react');
+import React from 'react';
+import getMuiTheme from 'material-ui/styles/getMuiTheme';
+import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
 
-var Search = React.createClass({
-  render: function () {
+import TextField from 'material-ui/TextField';
+import AutoComplete from 'material-ui/AutoComplete';
+
+let Search = React.createClass({
```

Comment:
Why are we instantiating the class in this way? Is it in order to use the getInitialState function?

class ExampleComponent extends React.Component { constructor() { super(); this. _handleClick = this. _handleClick.bind(this); this.state = Store.getState(); }

It looks like maybe we can just store the state in this.state